.takeshape/pattern/schema.json

```diff
@@ -668,7 +782,7 @@
       "serviceType": "rest",
       "authenticationType": "basic",
       "options": {
-        "endpoint": ""
+        "endpoint": "https://takeshape.gorgias.com/api/"
```

Comment:
Empty string options are filled since this schema is now used for previews and needs to work.

Reply:
Do the empty strings cause errors, or you just mean we need a functional contact form?

Reply:
It's to make it functional.